src/stock.rs

```diff
@@ -240,6 +240,10 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Chat protection disabled."))]
     ProtectionDisabled = 89,
+
+    // used in summaries, a noun, not a verb (not: "to reply")
+    #[strum(props(fallback = "Reply"))]
```

Comment:
Signal uses `(reply)` which I like because the brackets indicate that this is not an actual text
```suggestion
    #[strum(props(fallback = "(reply)"))]
```

Reply:
yip, i've seen that. but we are also not using "(image)" or so; i think it is fine without brackets as well.